### PR TITLE
[ADD] website_backend_views

### PR DESCRIPTION
**Pull request by hbrunn**
# Backend views for website

This module allows to embed old style (backend) views into the website frontend. This way, complex forms don't need to be redeveloped but still there's no break in user experience. Caveat is that this does not work for anonymous users.

This is specifically convenient if you need to use on_change functions, user input dependent domains or got used to be able to create linked objects inline.
# Usage

Install this module with demo data enabled and visit `/website_backend_views/demo`. Then read `view/demo.xml`.
# Known issues / Roadmap
- more testing needed
- not all css styles from the backend apply (should be just a matter ob having the right html structure/classes attached to the container
- maybe we can do something for anonymous users?
- view switching?
# Credits
## Contributors
- Holger Brunn hbrunn@therp.nl
## Maintainer

.. image:: http://odoo-community.org/logo.png
:alt: Odoo Community Association
:target: http://odoo-community.org

This module is maintained by the OCA.

OCA, or the Odoo Community Association, is a nonprofit organization whose mission is to support the collaborative development of Odoo features and promote its widespread use.

To contribute to this module, please visit http://odoo-community.org.

### CODE REVIEW

File: website_backend_views/README.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
+Backend views for website
+=========================
+
+This module allows to embed old style (backend) views into the website frontend. This way, complex forms don't need to be redeveloped but still there's no break in user experience. Ceveat is that this does not work for anonymous users.
```
Comment (pedrobaeza) [inline website_backend_views/README.rst]: s/Ceveat/Caveat/

File: website_backend_views/README.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Usage
+=====
+
+Install this module with demo data enabled and visit `/website_backend_views/demo`. Then read `view/demo.xml`.
+
+Known issues / Roadmap
+======================
+
+* more testing needed
+* not all css styles from the backend apply (should be just a matter ob having the right html structure/classes attached to the container
```
Comment (pedrobaeza) [inline website_backend_views/README.rst]: s/ob/of/ and please close parenthesis

File: website_backend_views/README.rst [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+* maybe we can do something for anonymous users?
```
Comment (pedrobaeza) [inline website_backend_views/README.rst:18]: With portal permissions I think it's enough. Don't you?

**Comment by pedrobaeza** [comment]
Please remove Therp logo

**Comment by hbrunn** [comment]
Thanks Pedro, you review faster than I can notice that I didn't clean up the logo from my default template ;-)

File: website_backend_views/README.rst (hunk 3 above)
Comment (hbrunn) [inline website_backend_views/README.rst:18]: Yes, but that's a user. The whole backend framework needs a uid, but I think we could create one without permissions (~portal) and su all calls to that. But it's more tricky than it sounds I think

Comment (pedrobaeza) [inline website_backend_views/README.rst:18]: Yeah, very tricky. A use case that I can imagine for this is to replace a form to fill by a backend view, but this have to be done as a very special mode with user notice I think.

**Comment by guewen** [comment]
Wow
LGTM (code review, no test)

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**